hdf/src/tbbt.c

```diff
@@ -649,16 +649,17 @@ tbbtrem(TBBT_NODE **root, TBBT_NODE *node, VOIDP *kp)
 TBBT_TREE *
 tbbtdmake(intn (*cmp)(VOIDP /* k1 */, VOIDP /* k2 */, intn /* arg */), intn arg, uintn fast_compare)
 {
-    TBBT_TREE *tree = Alloc(1, TBBT_TREE);
-
+    TBBT_TREE *tree = malloc(sizeof(TBBT_TREE));
     if (NULL == tree)
-        return (NULL);
+        return NULL;
+
     tree->root         = NULL;
     tree->count        = 0;
     tree->fast_compare = fast_compare;
     tree->compar       = cmp;
     tree->cmparg       = arg;
-    return (tree);
+
+    return tree;
 }
 
 #ifdef WASTE_STACK
```

Comment:
This ifdef seems useless now.

Reply:
There are still a bunch of useless ifdefs. We'll address them as time goes on.